src/components/BackgroundImageText/index.css

```diff
@@ -0,0 +1,34 @@
+@import "../../styles/all.css";
+
+.root {
+  position: relative;
+
+  display: inline-block;
+
+  color: $lavender;
+  -webkit-background-clip: text;
+
+  -webkit-text-fill-color: transparent;
```

Comment:
Why not use `color: transparent`? This will only work on Chrome based devices.

Reply:
Carefull using this, we used it on the Noken website but we knew the consequences. `webkit` basically means Chrome and Safari only